cmd/openshift-install/gather.go

```diff
@@ -217,18 +217,26 @@ func logClusterOperatorConditions(ctx context.Context, config *rest.Config) erro
 
 	for _, operator := range operators.Items {
 		for _, condition := range operator.Status.Conditions {
-			if condition.Type == configv1.OperatorUpgradeable {
-				continue
-			} else if condition.Type == configv1.OperatorAvailable && condition.Status == configv1.ConditionTrue {
+			switch condition.Type {
+			case configv1.OperatorAvailable:
+				if condition.Status == configv1.ConditionTrue {
+					continue
+				}
+			case configv1.OperatorDegraded:
+				if condition.Status == configv1.ConditionFalse {
+					continue
+				}
+				logrus.Errorf("Cluster operator %s %s is %s with %s: %s", operator.ObjectMeta.Name, condition.Type, condition.Status, condition.Reason, condition.Message)
```

Comment:
I know it's not the point of this PR, but I wish we could make these messages read a bit better. The results are close to actual sentences but the grammar doesn't work and I think that is distracting.

 What about `"%s cluster operator's %s status is %s with %s: %s"`

| Current  | Suggestion |
| ------------- | ------------- |
| `Cluster operator console Progressing is True`   | `Console cluster operator's Progressing status is True`  |
| `Cluster operator console Available is False` | `Console cluster operator's Available status is False`  |

Reply:
> Console cluster operator's Progressing status is True

We can't do "Console" without logic to capitalize names, and I think case-munging makes grepping harder.  We could do:

```
Cluster operator console has Progressing=True with SyncLoopRefreshProgressingInProgress: SyncLoopRefreshProgressing: Working toward version 4.3.0-0.ci-2019-11-01-0703
```

and similar.  Although even more tangentially that console message stutters the reason.

Reply:
> We can't do "Console" without logic to capitalize names
Ah, ok. Didn't consider that. You could use the same approach to mitigating this problem in my suggestion as you used in yours--although "cluster operator" as adjective should be "cluster-operator" but perhaps that is pedantic and distracting.  Regardless I like your suggestion and see it as an improvement. 

> Although even more tangentially that console message stutters the reason.
How do you mean? 

I think the issue is e.g. `Cluster operator console Available`, which seems like a jumble of nouns, is difficult to untangle.

Reply:
I'm fine with:

```go
logrus.Errorf("Cluster-operator %s has %s=%s with %s: %s", operator.ObjectMeta.Name, condition.Type, condition.Status, condition.Reason, condition.Message)
```

@abhinavdahiya, if that sounds good to you too, let me know if you want it here or in a separate PR.

And I meant `SyncLoopRefreshProgressingInProgress: SyncLoopRefreshProgressing:` stutters, but that's the console, not us.